numpy/f2py/tests/test_return_integer.py

```diff
@@ -36,7 +36,7 @@ def check_function(self, t):
         assert_raises(Exception, t, t)
         assert_raises(Exception, t, {})
 
-        if t.__doc__.split()[0] in ['t8', 's8']:
```

Comment:
I assume `t.__name__` doesn't work here?

Reply:
I didn't try, I just adopted the `tname` form from elsewhere in the tests

Reply:
Tried it, `__name__` is not set on these `<fortran>` objects